### PR TITLE
ref(replay): add resource about identifying users in configure card

### LINKED ISSUE #70276: [Design] Proactively Show Customers How They Can Change their Replay Identifier from IP Address
Today, many projects collect IP addresses and those are used to identify replay sessions unless `Sentry.setUser` is specified.

We want a way to indicate to customers how they can use a more friendly user-specific identification such as email, user name or user ID instead: https://docs.sentry.io/platforms/javascript/session-replay/configuration/#identifying-users

### PR DESCRIPTION
**Pull request by michellewzhang**
SCR-20241212-lxhq

closes https://github.com/getsentry/sentry/issues/70276

### CODE REVIEW

**Review by c298lee — APPROVED** [review]

File: static/app/components/replays/configureReplayCard.tsx [hunk 1]
```diff
@@ -54,6 +54,11 @@ function ResourceButtons() {
         subtitle={t('Unmask text (****) and unblock media (img, svg, video, etc.)')}
         link="https://docs.sentry.io/platforms/javascript/session-replay/privacy/#privacy-configuration"
       />
+      <Resource
+        title={t('Identify Users')}
+        subtitle={t('Identify your users through a specific format, such as email.')}
```
Comment (billyvg) [inline static/app/components/replays/configureReplayCard.tsx:59]: Not sure "format" is the right word here...

Maybe attribute? property? Don't love those either...

**Review by billyvg — APPROVED** [review]

File: static/app/components/replays/configureReplayCard.tsx (hunk 1 above)
Comment (michellewzhang) [reply static/app/components/replays/configureReplayCard.tsx:59]: hm @jas-kas do you have a pref? attribute & property are both fine w me

Comment (Jesse-Box) [reply static/app/components/replays/configureReplayCard.tsx:59]: Attribute sounds good to me

Comment (michellewzhang) [reply static/app/components/replays/configureReplayCard.tsx:59]: thanks @Jesse-Box ! i'll update

**Pull request merged**